source/macos/AdaptiveCards/AdaptiveCards/Managers/ACSFillerSpaceManager.swift

```diff
@@ -8,7 +8,9 @@ import AdaptiveCards_bridge
 import AppKit
 
 let kFillerViewLayoutConstraintPriority = NSLayoutConstraint.Priority.defaultLow - 10
-let kStrechableViewLayoutConstraintPriority = NSLayoutConstraint.Priority.defaultLow + 10
+
+class StretchableView: NSView {
+}
```

Comment:
What is the point of this?

Reply:
we can easily detect stretchable view from view hierarchy.